.github/workflows/terratest.yml

```diff
@@ -0,0 +1,47 @@
+name: Run Tests
+
+
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+
+jobs:
+  go-tests:
+    name: Run Go Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-go@v1
+        with:
+          go-version: 1.18
+      - uses: autero1/action-terraform@v0.1.0
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: true
+        with:
+          terraform_version: 1.0.11
+      - name: install wg and yggdrasil and add peers
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y wireguard
+          sudo apt-get install dirmngr
+          gpg --fetch-keys https://neilalexander.s3.dualstack.eu-west-2.amazonaws.com/deb/key.txt
+          gpg --export 569130E8CA20FBC4CB3FDE555898470A764B32C9 | sudo apt-key add -
+          echo 'deb http://neilalexander.s3.dualstack.eu-west-2.amazonaws.com/deb/ debian yggdrasil' | sudo tee /etc/apt/sources.list.d/yggdrasil.list
+          sudo apt-get update
+          sudo apt-get install yggdrasil
+          sudo systemctl enable yggdrasil
+          sudo  sed -i -- 's/Peers\: \[\]/Peers\: \[\n\ttcp\:\/\/51\.15\.204\.214\:12345\n\ttcp\:\/\/94\.130\.203\.208\:5999\n\]/g'  /etc/yggdrasil.conf
```

Comment:
Should the peer list be fetched from [here](https://raw.githubusercontent.com/threefoldtech/zos-config/main/development.json) instead? They are the peer list used by the zos nodes and they are constantly being updated. If for some reason the hardcoded peers went down, it would make the tests fail.